examples/src/main/kotlin/org/dashj/platform/examples/GetNetworkStatus.kt

```diff
@@ -165,7 +165,7 @@ class GetNetworkStatus {
 
             println("${badNodes.size} masternodes failed to respond to one or more requests")
             println("List of non-responsive nodes:\n $badNodes")
-            println("\nList of nodes with no errors:\n\t${goodNodes}\n")
+            println("\nList of nodes with no errors:\n\t${goodNodes.map { "\"$it\"" }}\n")
```

Comment:
this change will allow us to take the list of good nodes and copy/paste it into `TestNet3Params` in `dashj`